plugins/woocommerce/client/blocks/assets/js/previews/product-response-items.ts

```diff
@@ -0,0 +1,236 @@
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { WC_BLOCKS_IMAGE_URL } from '@woocommerce/block-settings';
+import { ProductResponseItem } from '@woocommerce/types';
+
+export const previewProductResponseItems: ProductResponseItem[] = [
+	{
+		id: 1,
+		name: __( 'Beanie', 'woocommerce' ),
+		parent: 0,
+		type: 'simple',
+		variation: '',
+		permalink: 'https://example.org',
+		slug: 'wp-beanie',
+		sku: 'wp-beanie',
+		short_description: __( 'Warm hat for winter', 'woocommerce' ),
+		description:
+			'Pellentesque habitant morbi tristique senectus et netus et malesuada fames ac turpis egestas. Vestibulum tortor quam, feugiat vitae, ultricies eget, tempor sit amet, ante. Donec eu libero sit amet quam egestas semper. Aenean ultricies mi vitae est. Mauris placerat eleifend leo.',
+		price: '7.99',
+		price_html:
+			'<span class="woocommerce-Price-amount amount"><span class="woocommerce-Price-currencySymbol">£</span>7.99</span>',
+		images: [
+			{
+				id: 1,
+				src: WC_BLOCKS_IMAGE_URL + 'previews/beanie.jpg',
+				thumbnail: WC_BLOCKS_IMAGE_URL + 'previews/beanie.jpg',
+				name: 'beanie-1.jpg',
+				alt: 'Beanie',
+				srcset: '',
+				sizes: '',
+			},
+		],
+		average_rating: '5',
+		categories: [
+			{
+				id: 1,
+				name: 'Clothing',
+				slug: 'clothing',
+				link: 'https://example.org',
+			},
+		],
+		tags: [],
+		attributes: [],
+		variations: [],
+		review_count: 1,
+		prices: {
+			currency_code: 'GBP',
+			currency_symbol: '£',
+			currency_minor_unit: 2,
+			currency_decimal_separator: '.',
+			currency_thousand_separator: ',',
+			currency_prefix: '£',
+			currency_suffix: '',
+			price: '7.99',
```

Comment:
Minor but these should be in smallest unit, no? 🤔 And then, they are adjusted for rendering on the frontend using `currency_minor_unit`. Otherwise we have super small unrealistic prices 😅 

```suggestion
			price: '799',
```

<img width="514" height="335" alt="image" src="https://github.com/user-attachments/assets/fa561a14-8ecc-4b1d-b394-54f8efec3659" />

If you agree it should be applied to the all price values.

Reply:
Done in e87df57b22dbbd41ba53c8d217c01ecfcf6616ec.